### ISSUE TITLE
Повторяющаяся строчка кода

### ISSUE THREAD

**Issue by Stepa86** [1c-syntax/bsl-language-server#284]
если один и тот же код написан 2 раза подряд, это очень подозрительно

```bsl
Перем1 = СуперРезультат();
Перем1 = СуперРезультат();

СделатьХорошо();
СделатьХорошо();
СделатьЕщеХорошо();

Если УсловиеВыполнено Тогда
    ВыполнитьДвижуху();
КонецЕсли;

Если УсловиеВыполнено Тогда
    ВыполнитьДвижуху();
КонецЕсли;
```

**Comment by nixel2007** [comment]
мм... это вроде делает CPD в сонаре?

**Comment by Stepa86** [comment]
А он ишуз повесит на это?

**Comment by nixel2007** [comment]
Да. Возможно не на такой маленький блок кода, но вообще ишузы на копипасту есть

**Comment by Stepa86** [comment]
У меня проигнорировал

![image](https://user-images.githubusercontent.com/25096392/61975674-73210100-aff2-11e9-95b3-7460b1d5f66f.png)

**Comment by nixel2007** [comment]
Там пороги шибко больше. Кажется, от 70 токенов

**Comment by Stepa86** [comment]
Эта проверка про ошибку копипасты, когда во второй строке должна была быть другая переменная или другой метод, но продолбалось. Вот в статье про это https://habr.com/ru/company/pvs-studio/blog/461519/

А сам сонар ловит дублирование - когда этот кусок стоило бы вынести в отдельный метод
